capturebody.go

```diff
@@ -70,13 +70,19 @@ func (t *Tracer) CaptureHTTPRequestBody(req *http.Request) *BodyCapturer {
 		io.Reader
 		io.Closer
 	}
+	var err error
+	copiedBody := req.Body
+	copiedBody, req.Body, err = drainBody(req.Body)
```

Comment:
If the client streams its body, this will cause the middleware to block until the whole body is received. I don't think that's desirable behaviour. This is the reason why we don't eagerly read the body, but instead wrap the body so that it is tee'd.